src/table/__integ__/sticky-columns.test.ts

```diff
@@ -0,0 +1,209 @@
+  const isOverflowing = await browser.execute((containerSelector: string) => {
+    const container = document.querySelector(containerSelector) as HTMLElement;
+    return container.scrollWidth > container.clientWidth;
+  }, selector);
+
+  return isOverflowing;
+}
+
+let rowCount: number;
+let columnCount: number;
+
+const checkStickyCells = async (
+  browser: any,
+  wrapper: TableWrapper,
+  columnIndex: number,
+  rowCount: number,
+  shouldBeSticky: boolean
+) => {
+  // Expect header cell to be sticky
+  const headerCellSelector = wrapper.findColumnHeaders().get(columnIndex).toSelector();
+  const position = await browser.execute(getElementComputedStyle, headerCellSelector, 'position');
+  await (shouldBeSticky ? expect(position) : expect(position).not).toEqual('sticky');
+
+  // Expect all body cells in the column to be sticky
+  for (let i = 1; i <= rowCount; i++) {
+    const cellSelector = wrapper.findBodyCell(i, columnIndex).toSelector();
+    const position = await browser.execute(getElementComputedStyle, cellSelector, 'position');
+    await (shouldBeSticky ? expect(position) : expect(position).not).toEqual('sticky');
+  }
+};
+
+async function navigateToTestPage(browser: any, dataTestId = 'simple', queryParams = '') {
+  await browser.url(`#/light/table/sticky-columns${queryParams}`);
+  // Simple table has stickyColumns: {first: 1, last: 1}
+  const wrapper = createWrapper().findTable(`[data-test-id='${dataTestId}']`);
+  const page = new StickyColumnsPage(browser);
+  await page.waitForVisible(wrapper.findRows().toSelector());
+
+  rowCount = await page.getElementsCount(wrapper.findRows().toSelector());
+  columnCount = await page.getElementsCount(wrapper.findColumnHeaders().toSelector());
+
+  return { page, wrapper };
+}
+
+test(
+  'first and last columns are sticky',
+  useBrowser(async browser => {
+    const { wrapper } = await navigateToTestPage(browser);
+
+    await checkStickyCells(browser, wrapper, 1, rowCount, true);
+    await checkStickyCells(browser, wrapper, 2, rowCount, false);
+    await checkStickyCells(browser, wrapper, columnCount - 1, rowCount, false);
+    await checkStickyCells(browser, wrapper, columnCount, rowCount, true);
+  })
+);
+
+test(
+  'shadows are displayed correctly',
+  useBrowser(async browser => {
+    const { page, wrapper } = await navigateToTestPage(browser);
+
+    // Before scroll there is no shadow on the first column
+    await forEachColumnCell(wrapper, 1, async cellSelector => {
+      const boxShadow = await browser.execute(getElementComputedStyle, cellSelector, 'boxShadow');
+      await expect(boxShadow).toBe('none');
+    });
+
+    // Before scroll there is shadow on the last column
+    await forEachColumnCell(wrapper, columnCount, async cellSelector => {
+      const boxShadow = await browser.execute(getElementComputedStyle, cellSelector, 'boxShadow');
+      await expect(boxShadow).toBe('rgba(0, 28, 36, 0.1) -4px 0px 20px 1px');
```

Comment:
Do we need to be that specific? Wouldn't it be enough to assert the presence of the shadow?